build/ansible/roles/pmm-images/tasks/main.yml

```diff
@@ -82,11 +93,11 @@
         comment: "{{ item.comment }}"
         shell: "{{ item.shell }}"
         group: "{{ item.group }}"
-        non_unique: true
       loop:
         - { name: pmm, uid: 1000, comment: "PMM Server", shell: "/usr/bin/bash", home: "/home/pmm", group: pmm, }
-        - { name: nginx, uid: 999, comment: "nginx user", shell: "/sbin/nologin", home: "/dev/null", group: nginx, }
-        - { name: clickhouse, uid: 997, comment: "Clickhouse server", shell: "/sbin/nologin", home: "/var/lib/clickhouse", group: clickhouse, }
+        - { name: nginx, uid: 1002, comment: "nginx user", shell: "/sbin/nologin", home: "/dev/null", group: nginx, }
```

Comment:
These used to be created by the respective package installs. We create them only because we don't want to see uid/gid under 1000. They get removed anyway in post-build.yml.